repos/system_upgrade/el7toel8/actors/checksaphana/libraries/checksaphana.py

```diff
@@ -0,0 +1,160 @@
+from leapp.libraries.stdlib import api
+from leapp.libraries.common.config import architecture
+from leapp.models import SapHanaInfo
+from leapp import reporting
+
+
+# SAP HANA Compatibility
+
+SAP_HANA_MINIMAL_MAJOR_VERSION = 2
+SAP_HANA_MINIMAL_SP = 5
+SAP_HANA_MINIMAL_REV = 52
+SAP_HANA_MINIMAL_VERSION_STRING = 'HANA 2.0 SPS05 rev 52'
+
+
+def running_check(info):
+    if info.running:
+        reporting.create_report([
+            reporting.Title('Found running SAP HANA instances'),
+            reporting.Summary(
+                'In order to perform a system upgrade it is necessary that all instances of SAP Hana are stopped.'
+                'Please shutdown all SAP Hana instances before you continue with the upgrade.'
+            ),
```

Comment:
the instruction should be part of remediation hint. no big problem but i guess it would be better if we put this stuff into the remediation as hint always. Some people would go for remediations mainly during reading.